src/pages/Main/Main.scss

```
@@ -0,0 +1,7 @@
+.carousel {
+  display: block;
+  margin: 10px auto 50px auto;
```

```suggestion
  margin: 10px auto 50px ; 
```
margin, padding에 속성을 부여할 경우 상황에 따라서 shorthand property를 이용해서 손쉽게 부여할 수 있습니다.

---

수정하였습니다.